CONTRIBUTING.md

```diff
@@ -4,23 +4,43 @@
 
 We use [pnpm](https://pnpm.io) to manage our dependencies.  
 You can [install it](https://pnpm.io/installation) using your preferred
-method, but we recommend using [Corepack](https://nodejs.org/api/corepack.html#supported-package-managers) so that you
-keep your package manager version in sync with ours.
+method, but we recommend using [Corepack](https://nodejs.org/api/corepack.html#supported-package-managers) so that you keep your package manager version in sync with ours.
 
 Make sure you run **Node 20** or the dependencies' installation will fail.  
```

Comment:
we must use node 20.8.1.
Latest version of node don't work.

Reply:
We can add this text : 

"Be sure to use the latest version of pnpm and node 20.8.1"